ADO-HW/ADO-HW/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ADO_HW
+{
+    class Program
+    {
+        private static readonly string ConnectionString =
+            @"Server=DESKTOP-D5VI0OE;Database=Northwind;Trusted_Connection=True;";
+
+        static void Main(string[] args)
+        {
+            var totalCountOrders = SalesByYear(new DateTime(1997, 01, 01), new DateTime(1998, 01, 01)).Count;
+
+            var totalPriceOfMostExpensiveTenProducts = TenMostExpensiveProducts()
+                                                       .Select(_ => _.UnitPrice)
+                                                       .Sum();
+        }
+
+        private static List<Order> SalesByYear(DateTime BeginningDate, DateTime EndingDate)
+        {
+            var result = new List<Order>();
+            
+            string storedProcedure = @"Sales by Year";
```

Comment:
As far as I remember this stored procedure wasn't created by you but in that case, the name of the stored procedure should include the action it does (e.g., GetSalesByYear)